src/templates/reset-password.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import {
+  MjmlColumn,
+  MjmlText,
+  MjmlSpacer,
+  MjmlImage,
+  MjmlButton,
+} from 'mjml-react';
+
+import * as Colors from '../constants/colors';
+import { getAssetPath } from '../utils/url';
+
+import { NarrowContentSection } from '../components/content';
+import { AppCTASection } from '../components/cta-app';
+import { FooterSection } from '../components/footer';
+import { HeaderSection } from '../components/header';
+import { BasicSection } from '../components/sections/basic';
+import { TemplateBase } from '../components/template-base';
+
+export interface ResetPassTemplateProps {
+  title?: string;
+  titleTop?: React.ReactElement;
+  subtitle?: string;
+  subtitlePrepend?: React.ReactElement;
+  avatarURL?: string;
+  isCivicLiker?: boolean;
+  content?: string;
+  language?: string;
+}
+
+export const ResetPassTemplate = ({
+  title,
+  titleTop,
+  subtitle,
+  subtitlePrepend,
+  content,
+  language,
+}: ResetPassTemplateProps) => {
+  return (
+    <TemplateBase language={language} isExtruded={true}>
+      <HeaderSection />
+      <BasicSection
+        paddingTop={0}
+        paddingBottom={0}
+        paddingLeft={0}
+        paddingRight={0}
+        backgroundUrl={getAssetPath('/templates/basic/header-top.jpg')}
+      >
+        <MjmlColumn width={120} paddingLeft={0} paddingRight={0}>
+          {titleTop || (
+            <MjmlImage
+              src={getAssetPath('/security.png')}
+              width={120}
+              height={120}
+              borderRadius={60}
+            />
+          )}
+        </MjmlColumn>
+      </BasicSection>
+      <BasicSection
+        backgroundUrl={getAssetPath('/templates/basic/header-middle.jpg')}
+        paddingTop={16}
+        paddingBottom={16}
+      >
+        <MjmlColumn>
+          {title && (
+            <MjmlText align="center" fontSize={24}>
+              {title}
```

Comment:
I think title can be hardcoded for reset password template

Reply:
do you mean "Reset Password Instruction"? it's actually a subtitle xD